shared/templates/grub2_bootloader_argument/ansible.template

```diff
@@ -4,5 +4,10 @@
 # complexity = medium
 # disruption = low
 
+{{% if ARG_VARIABLE %}}
+{{{ ansible_instantiate_variables(ARG_VARIABLE) }}}
+{{% set ARG_NAME_VALUE= ARG_NAME ~ "=\"{{" ~ ARG_VARIABLE ~ "}}\""%}}
```

Comment:
There is an GitHub Actions CI warning in https://github.com/ComplianceAsCode/content/runs/5422661136?check_suite_focus=true: `Warning: var-spacing Variables should have spaces before and after: --args="slub_debug="{{var_slub_debug_options}}""`

Reply:
I also think that one can use " and ' quoting [interchangeably](https://jinja.palletsprojects.com/en/3.0.x/templates/#literals) in jinja2, and I would advise to mix them in order to avoid escaping of quotes.